TeamCode/src/main/java/suitbots/GyroTest.java

```diff
@@ -14,7 +14,10 @@
     @Override
     public void init() {
         robot = new Robot(hardwareMap);
-
+        while (!robot.isGyroCalibrated()) {
```

Comment:
Code in `init()` can't block. By "block" I mean "do anything that takes more than a very short period of time. Waiting for calibration to finish counts.

Instead, for a non-Linear `OpMode`, you can do this sort of stuff in a function called `@Override public void init_loop()`.  But that shouldn't block, either, so instead you'd just want to check to see if it's calibrated once, set the appropriate telemetry data and then update telemetry.

Reply:
This one still definitely needs to change. The OpMode will likely crash (a timeout in `init`) if it's done this way.